### ISSUE TITLE
can you give instruction to step by step jailbreak?

### ISSUE THREAD

**Issue by pro100saxarok** [zhengmin1989/iOS_ICE_AND_FIRE#5]
can you give instruction to step by step jailbreak? (ex. 1.Take phone 2. Take cable. 3. Connect Phone to PC... etc.) ?) Yes, im idiot ;D

**Comment by Ethicalgamer** [comment]
I am in need of the instructions too. Can someone please help rather than makin fun ?

**Comment by dalisoft** [comment]
I need too this damn instruction

**Comment by miayotlan** [comment]
I'm not quite sure but I believe u have to download the project compile it and sideload it with Xcode and then execute from your compatible idevice it might be helpful too if you watch the demo video so u know what to type on the terminal also Google search how to install cydia from terminal the following link might help you with instructions how to get cydia from mobile terminal - - ->http://goo.gl/5IBe3w. Again I'm not sure if this is how u use it but it makes sense to sideload it with Xcode and execute it from ur device unfortunately I don't have a compatible idevice mine is running 9.2.1 but is an iPhone 6 Plus so is a arm64  not a 32 so I'm gonna wait to see if someone gets this working with arm64  I hope this was some how helpful  :)

**Comment by dalisoft** [comment]
Thanks @miayotlan and you is amazing

**Comment by miayotlan** [comment]
No problem hopefully that works for u here's also a list of mobile terminal commands.  
https://goo.gl/W9YDVa

**Comment by warkrave** [comment]
Sorry @miayotlan but i don't understand your explanation... We have to download Xcode on mac ?

**Comment by dalisoft** [comment]
Hi @warkrave 
1. Download XCode.
2. Open `.xproj` with XCode.
3. Debug it / Sideload.

NOTE: You can install XCode on macOS or on Windows via Virtual Machine

**Comment by GrumpyOet** [comment]
wich .xproj do i have need to open

**Comment by GrumpyOet** [comment]
where is it located

**Comment by zffs** [comment]
could you upload the compiled thing dalisoft? thanks

**Comment by GrumpyOet** [comment]
yeah can u do that

**Comment by zffs** [comment]
grumpy you have to execute the makefile to create the xproj files but you cant do that if you are not on a mac

**Comment by GrumpyOet** [comment]
if i do that i get this make: Nothing to be done for `all'.

i am on a mac

**Comment by zffs** [comment]
hmm

**Comment by GrumpyOet** [comment]
what do i have to do now

**Comment by GrumpyOet** [comment]
reinstall make or something

**Comment by zffs** [comment]
wait lmao

**Comment by GrumpyOet** [comment]
?

**Comment by zffs** [comment]
the "hello " executeable

**Comment by zffs** [comment]
you dont need to use the makefile

**Comment by zffs** [comment]
theres already a compiled file in there

**Comment by GrumpyOet** [comment]
what do i have need to do with the hello file

**Comment by zffs** [comment]
put it in xcode and sideload it something

**Comment by GrumpyOet** [comment]
do i have need to create a new project and put it in there and click on run

**Comment by GrumpyOet** [comment]
how do i sideload that hello file

**Comment by dalisoft** [comment]
extract zip, and there have `.xproj` file and open it with XCODE. My PC not a Mac. So i fail my thing. But this jailbreak worked for my friend iPhone 5S iOS 9.3.2 and he already runned Cydia. Someone have Mac? Pls help me. Thanks

**Comment by GrumpyOet** [comment]
i runed hook1 and what now

**Comment by GrumpyOet** [comment]
how do i install the hello file

**Comment by jmukes97** [comment]
I have tried Downloading the tool, then unzipping the ice and fire project folder. Then opening the project file there but i get the error

clang: error: no such file or directory: '/Users/nessam/Desktop/iOS_ICE_AND_FIRE/tools/objpwn/talker.m'
clang: error: no input files

So then i went to the iOS ice and fire folder, and created a folder called tools then objpwn and put all the files from Objective-C_Pwn into the objpwn folder. Then moved the entire thing to my desktop. The build sucseeded. When i opened the side loaded app on my phone all i got was a white app. No buttons no graphics nothing. With Xcode i saw this for the output:

2016-06-20 13:37:51.851 IceAndFire[2250:844641] Hello, Ice and Fire!
2016-06-20 13:37:51.853 IceAndFire[2250:844641] Hello, Ice and Fire!
(lldb) 

So i am pretty much stuck here. I got the app on my iPhone 5 running iOS 9.2.1 but it doesnt look like the app in the demo video. It looks like an app with no icon. I also unzipped the hook1 folder and opened the project file i found in there too. I Couldn't get this to run or even build to put it on my iPhone. i got this error:

CodeSign error: code signing is required for product type 'Dynamic Library' in SDK 'iOS 9.3'

Im pretty sure the last error is simple to fix. It looks like i just need to change a few settings? Im not all that familiar with Xcode so I'm not exactly sure. I also extracted the IOHDFamily folder and there was a tools folder there. There was also a project file there but i didn't try to put that into Xcode. Im still experimenting but if someone finds the answer let me know please!

**Comment by dalisoft** [comment]
I recommend watch guide video from YouTube as i used them every day. Thanks and sorry for can't help, because i in Win7 and never used nor Mac or xCode

**Comment by djelsmf** [comment]
jmukes97, which Xcode of u use?

**Comment by jmukes97** [comment]
Dalisoft can you give me a link to a video tutorial? I couldn't find one. Thanks!

Djelsmf I am using Xcode 7.3.1. 

Also I am still pretty new to Xcode, but I am familiar with programming. When i was looking through the folders i noticed that both of the storyboard files were completely blank. Aren't they supposed to have the app layout? Or am I misunderstanding?

**Comment by dalisoft** [comment]
@jmukes97
http://stackoverflow.com/questions/7486326/build-run-ios-xcode-project-from-terminal
https://www.youtube.com/watch?v=1IULUVV2MV8
https://www.youtube.com/watch?v=01eGo6YSPVc
https://www.youtube.com/watch?v=2-NcJMPNgU8
https://www.udemy.com/apple-mobile-app-development-with-swift-xcode-and-ios/

UPDATE 1: https://ipswcentral.com/news/ios-9-2-1-jailbreak-released/

I think it's video's will be useful for you. Not sure, but thanks for trying some work for we.

**Comment by djelsmf** [comment]
jmukes97 how is it going?

**Comment by jmukes97** [comment]
Not well. I have been looking through the files and the story board files are blank. I don't think this project is complete. Even if you do compile it correctly you will just have a blank app, if I'm not mistaken

**Comment by miayotlan** [comment]
What idevice r u trying to jailbreak? Bc if is not compatible then most likely is not gonna run also when u sideload and app u have to trust it in the settings -general- profile-and trust the app in order for it to work

**Comment by miayotlan** [comment]
For those of u who ask the compile file to be upload it u mean the .ipa? It wouldn't work bc it needs to be sign in order to create it and we all have different apple id's so it wouldn't work unless it was compile to a .deb file so everyone could download it and sign with the this tool  https://github.com/DanTheMan827/ios-app-signer which is use to sign and resign apps to sideload to ur idevice
Here's a link to a sideload tutorial on how to use it to sideload apps. https://m.reddit.com/r/jailbreak/comments/49fg46/tutorial_how_to_resign_apps_with_ios_app_signer/
Hope this helps

**Comment by jmukes97** [comment]
I am using an iPhone 5 with iOS 9.2.1. I know it is compatable. And I do generally know how to sideload apps on my phone with Xcode. I just keep running into errors. The tool isn't a ipa or a deb file. If anyone can help me out the app on my phone that would be great. The biggest problem is that the app just looks blank with no icon

**Comment by djelsmf** [comment]
so, u stuck not on making ipa or deb file, but on app's blank with on icon?
the same problem in emul of iOS in Xcode (yesterday i installed hackintosh and try run ice and fire in xcode in emul). and there is no errors, just blank app

**Comment by fakeusername1029391323** [comment]
if someone would be willing to post the .ipa that would be really cool, I'm on Windows so I can't sideload via Xcode but what  I could do is install the .ipa using a third party program on Windows.

**Comment by JasonHardon** [comment]
Hi, How can I load iOS_ICE_AND_FIRE on a Windows PC? I don't have X-Code. & I think iOS 9.2.1 might require the latest version of X-Code running El Capitan. Does not accept anything lower as far as I know. iOS 9.2.1 is not compatible with earlier versions of X-Code.

**Comment by jmukes97** [comment]
Dalisoft There's no need to make it into a .deb or .ipa file since the project files are in the folder. You just compile it directly on your device. And yeah that's the biggest problem. There's no icon and it's just a blank app. As I said before I think it is because the storyboard files are completly blank. So all you will get is a blank white app.

**Comment by dalisoft** [comment]
This means u cant jailbreak device?  Thanks

**Comment by djelsmf** [comment]
jmukes97, oh.. and one question. do you have a developer apple id to sideload proj into device?

**Comment by Ethicalgamer** [comment]
Do we need to be a developer to sideload ? If so, is that possible only on Mac ?

**Comment by fakeusername1029391323** [comment]
What File do you sideload?

**Comment by blubtxt** [comment]
There is no working jailbreak in that files! There are some PoCs and unfinished Code.

**Comment by warkrave** [comment]
https://www.youtube.com/watch?v=OQNWHdcGq7s

**Comment by JasonHardon** [comment]
Does anyone here know how to make a sandbox escape for this?

**Comment by JasonHardon** [comment]
Pangu to release new jailbreak within the week. Might be for 9.3 or 10 beta
https://superphen.co/2016/07/01/pangus-one-more-thing-at-mosec-2016-ios-10-jailbreak/

**Comment by djelsmf** [comment]
This is only rumors as always, as we can see

**Comment by JasonHardon** [comment]
Obviously not a rumour as you can see now. http://www.pangu.io/?flag=cn
Still waiting on an explanation from the creator of how to make this 9.2.1 Jailbreak work. Even if instructions are in Chinese, I don't care but something and maybe a follow up with the software Ice and Fire Flying JB project started here. It's been over a month since it was released.

**Comment by djelsmf** [comment]
Yep, indeed. But nobody relesaed yet Flying JB for <=9.2.1 for 32 bit idevices.
D'you think author reading us?

**Comment by JasonHardon** [comment]
Let's face it we all got screwed, no one cares about 32-bit devices anymore and Pangu just turned thier backs on all of us to make things worse.

**Comment by JasonHardon** [comment]
I just got a brand new iPhone 5 just a few months ago, now I gotta throw it away.

**Comment by djelsmf** [comment]
There is no news about jb 32 bit, but this is not a reason to throw dev out. :) come down, everything will be ok :)

**Comment by djelsmf** [comment]
No one were tried to read the code of project? may be there are some comments there to help?

**Comment by dalisoft** [comment]
@djelsmf THIS IS FAKE

**Comment by djelsmf** [comment]
@dalisoft 4 real? why do you think so?

**Comment by dalisoft** [comment]
@djelsmf Because no one can done this project, even with XCode, with skills with XCode. What do you think, if no one done? I tried with windows, no luck, because XCode haven't yet for windows.
If you done with it, i then beleive it's real.

**Comment by dalisoft** [comment]
@djelsmf If this already done, anyone can upload it to internet. make lot money with link shortener. thanks.

**Comment by ghost** [comment]
If a find a step by step tutorial on how to use this I'll probably use it to downgrade since I have a CFW for 7.1.2 and jailbreak from there :D

**Comment by JasonHardon** [comment]
Guys the solution is to simply buy a cheap 64bit SE iphone and jailbreak it since it will probably be on a lower iOS version coming from the Apple factory. So do it while you still can. Use Pangu. Good luck everyone as this has solved my issue since no explanation or instructions were given for this Flying bullsh... I mean jailbreak

**Comment by robonxt** [comment]
damn... I don't have a Mac to test this... and about the empty app, perhaps that's where the sandbox thingy is supposed to be? or maybe the exploit is there but the guy didn't add the run command, wanting programmers to find out the secret?
